setup.rst

```diff
@@ -43,7 +43,9 @@ Running your Symfony Application
 
 On production, you should use a web server like Nginx or Apache
 (see :doc:`configuring a web server to run Symfony </setup/web_server_configuration>`).
-But for development, it's convenient to use the :doc:`Symfony PHP web server <setup/built_in_web_server>`.
+But for development, it's convenient to use the :doc:`Symfony PHP web server <setup/built_in_web_server>`. 
+Instructions of how to install the development web server are also shown via the command line when first 
```

Comment:
`of` -> `on` 
`via` -> `in` this sounds better to me but not sure tho
`when first creating a new project.` -> `when you first create a new project`

Not sure about real value of this note.